frontend/components/molecules/Form/FieldSection.js

```diff
@@ -42,7 +44,7 @@ const FieldSection = ({title, description='', type="textbox", required, onChange
                         theme.textInputHeight.small, 
                         theme.textInputHeight.medium, 
                         theme.textInputHeight.large]} 
-                        width={["90%", "90%", "600px"]}
+                        width={"600px"}
```

Comment:
Was this UI change made on purpose ? Does the alignment still appear good when you make the browser width narrow ?

Reply:
Yes I think this was intentional, alignment should be good